src/test/java/org/openstreetmap/atlas/checks/validation/tag/StreetNameIntegersOnlyCheckTestRule.java

```diff
@@ -0,0 +1,77 @@
+package org.openstreetmap.atlas.checks.validation.tag;
+
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.utilities.testing.CoreTestRule;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Edge;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Loc;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Node;
+
+/**
+ * Tests for {@link StreetNameIntegersOnlyCheck}
+ *
+ * @author bbreithaupt
+ */
+
+public class StreetNameIntegersOnlyCheckTestRule extends CoreTestRule
```

Comment:
Can you please add an example that should be flagged with some unsavory breaking whitespace characters separating numbers?

Reply:
Should have had that, but it looks like I typoed the change into `motorwayWithNonIntegerNameTagAtlas` instead of `motorwayWithIntegerNameTagAtlas`. That will be fixed in a sec.